test/qa/test_meshnet.py

```diff
@@ -182,9 +186,11 @@ def test_set_meshnet_off_when_logged_out(meshnet_allias):
     assert "You are not logged in." in ex.value.stdout.decode("utf-8")
 
 
+@pytest.mark.smoke
```

Comment:
what is the difference between smoke mark and core_meshnet mark, aren't all smoke tests supposed to be in the core set?